test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -137,6 +137,12 @@ def test_amplitude_to_db(self):
         waveform = get_whitenoise(sample_rate=sample_rate, duration=0.05, n_channels=2)
         self.assert_grad(transform, [waveform])
 
+    def test_melscale(self):
+        sample_rate = 8000
+        transform = T.MelScale(sample_rate=sample_rate)
+        waveform = get_whitenoise(sample_rate=sample_rate, duration=0.05, n_channels=2)
```

Comment:
The input to `MelScale` has to be spectrogram. Passing waveform here, the `MelScale` will work on a wrong dimension, so we need to generate spectrogram from waveform and set the proper parameters. 

Something like this.

```python
n_fft = 400
n_mels = n_fft // 2 + 1

transform = T.MelScale(sample_rate=sample_rate, n_mels=n_mels)
spec = get_spectrogram(
    get_whitenoise(sample_rate=8000, duration=0.05, n_channels=2),
    n_fft=n_fft, power=1)
```

Reply:
Thanks for the review. To be honest I was not sure about this one.